### PR TITLE
Reword admin index links

### PR DESCRIPTION
**Pull request by kaitlynmichaelredislabs**
Reworded links so they made sense without the surrounding text (according to Google Style Guide)

### CODE REVIEW

**Review by banker — APPROVED** [review]

File: content/rc/administration/_index.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ aliases: /rv/administration/
...
-- [Create]({{<relref "rc/administration/setup/create-database.md">}}), [delete]({{<relref "rc/administration/setup/delete-database.md">}}), and [edit]({{<relref "rc/administration/setup/edit-database.md">}}) databases
-- [Monitor]({{<relref "rc/administration/configuration/monitoring-performance.md">}}) your database performance
-- [Backup]({{<relref "rc/administration/configuration/backups.md">}}) your databases to remote storage locations
+- [Create a subscription]({{<relref "/rc/administration/setup/create-subscription.md">}}), [customize a subscription]({{<relref "rc/administration/setup/customize-pro.md">}}), and [edit your subscription]({{<relref "rc/administration/setup/edit-subscription.md">}}) 
+- [Create a database]({{<relref "rc/administration/setup/create-database.md">}}), [delete  a database]({{<relref "rc/administration/setup/delete-database.md">}}), and [edit  a database]({{<relref "rc/administration/setup/edit-database.md">}}) 
+- [Monitor your database performance]({{<relref "rc/administration/configuration/monitoring-performance.md">}}) 
+- [Backup  your databases]({{<relref "rc/administration/configuration/backups.md">}}) to remote storage locations
```
Comment (banker) [inline content/rc/administration/_index.md:14]:
```suggestion
- [Backup your databases]({{<relref "rc/administration/configuration/backups.md">}}) to remote storage locations
```

File: content/rc/administration/_index.md [hunk 2]
```diff
@@ -8,8 +8,8 @@ aliases: /rv/administration/
...
 
-- [Create]({{<relref "/rc/administration/setup/create-subscription.md">}}), [customize]({{<relref "rc/administration/setup/customize-pro.md">}}), and [edit]({{<relref "rc/administration/setup/edit-subscription.md">}}) your subscription
-- [Create]({{<relref "rc/administration/setup/create-database.md">}}), [delete]({{<relref "rc/administration/setup/delete-database.md">}}), and [edit]({{<relref "rc/administration/setup/edit-database.md">}}) databases
-- [Monitor]({{<relref "rc/administration/configuration/monitoring-performance.md">}}) your database performance
-- [Backup]({{<relref "rc/administration/configuration/backups.md">}}) your databases to remote storage locations
+- [Create a subscription]({{<relref "/rc/administration/setup/create-subscription.md">}}), [customize a subscription]({{<relref "rc/administration/setup/customize-pro.md">}}), and [edit your subscription]({{<relref "rc/administration/setup/edit-subscription.md">}}) 
+- [Create a database]({{<relref "rc/administration/setup/create-database.md">}}), [delete  a database]({{<relref "rc/administration/setup/delete-database.md">}}), and [edit  a database]({{<relref "rc/administration/setup/edit-database.md">}})
```
Comment (banker) [inline content/rc/administration/_index.md:12]:
```suggestion
- [Create a database]({{<relref "rc/administration/setup/create-database.md">}}), [delete  a database]({{<relref "rc/administration/setup/delete-database.md">}}), and [edit a database]({{<relref "rc/administration/setup/edit-database.md">}}) 
```

**Review by lanceleonard — APPROVED** [review]
LGTM

**Comment by bmansheim** [comment]
Some of these changes were already committed. I fixed the double spaces in another commit.

**Pull request closed without merging**